Makefile

```diff
@@ -54,3 +53,15 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+rpm:
+	mkdir -p dist
+	chmod 777 `pwd`/dist
+	cd packages && docker build -t calico-mesos-rpm-builder .
+	cd packages && docker run \
+	 -v $(WD)/dist:/opt/rpms \
+	 calico-mesos-rpm-builder bash -c 'rpmbuild -ba /root/calico-mesos.spec && \
+	 cp `find /root/rpmbuild/RPMS -type f` /opt/rpms'
```

Comment:
Its in `/root/rpmbuild/RPMS/x86_64/*.rpm`. I wasn't sure if we would ever have plans to support other architectures. Should I still remove the `find` command?